elt.js

```diff
@@ -23,6 +23,34 @@ function createApiActionFor(fuseaction){
 	};
 }
 
+function createDateApiActionFor(fuseaction){
+	return function(data, callDate, callback) {
+		let timestamp = {
+			timestamp: new Date().getTime()
+		};
+
+		let callURL = baseURL + fuseaction.replace("{}", data);
+		callURL = callURL.replace("[]", 
```

Comment:
Between JS and JQuery there's no way to format this with a DateFormat type thing?

Yeah, definitely glad I'm a backend dev.  ;)

Reply:
It was late and I just missed that there was one specific toString that does actually work. There is no date toString with a format, just a lot of default ones to do various formats but a lot of them are more human readable which is not useful for what I needed.